apps/views.py

```diff
@@ -447,9 +447,21 @@ def start_election(request):
 
     This function is only accessible to the staff or superuser.
     """
+    previous_election = NewElection.objects.all()
+    for election in previous_election:
+        if check_election_status(election) == "Ongoing":
+            messages.error(request, 'There are ongoing election.')
+            return redirect('election_list')
+
```

Comment:
There is no need to query all election to check one by one. Use Django built-in `filter`

```suggestion
    if NewElection.objects.filter(start_date__gte=timezone.now(), end_date__lt=timezone.now()).exists():
            messages.error(request, 'There are ongoing election.')
            return redirect('election_list')
```